modules/ols-creating-lightspeed-custom-resource-file-using-cli.adoc

```diff
@@ -40,10 +40,57 @@ spec:
     defaultProvider: myOpenai
 ----
 +
-[NOTE]
-====
-For OpenShift AI vLLM use the same configuration as OpenAI but update the URL to point to your Virtual Large Language Model (vLLM) endpoint. If OpenShift Lightspeed operates in the same cluster as the vLLM model serving instance, you can point to the internal OpenShift service hostname instead of exposing vLLM with a route.
-====
+.Example {rhelai} custom resource file
++
```

Comment:
A `+` continuation is not needed between the `.` block title and the starting `----` code block itself.
```suggestion

```

Reply:
If it won't break anything I like it to help make seeing the chunks in the source a little easier.